tutorials/cp-mobile-dev-kit-cbs-client/cp-mobile-dev-kit-cbs-client.md

```diff
@@ -0,0 +1,349 @@
+---
+title: Build your Mobile Development Kit Client using Cloud Build Service
```

Comment:
Build a Mobile Development Kit Client Using Cloud Build Service

Reply:
I have updated it to : **Build Your Mobile Development Kit Client Using Cloud Build Service**
(to keep it similar to "Build Your Mobile Development Kit Client Using MDK SDK").